.github/ISSUE_TEMPLATE/Standard-Bug-Issue-Template.md

```diff
@@ -41,24 +41,21 @@ devchat: < If Hl Engineering requires notification instead of the end user or if
 < list of subject domain experts for this domain. If the subject expert from the end-user perspective is not the author, an end-user expert should be listed here. The Design, Dev, and QA teams should add experts here as appropriate. >
 
 ## Progress
-### Ready for Planning
+### Reviewed Issues - completed by SM & Team Leads
```

Comment:
I've been checking these boxes as soon as I write the issue before it hits grooming. Is that incorrect?

Reply:
You are not correct. I need 2 sets of checkboxes, one for the author and one for the SME Review board that cover basically the same things because sometimes the author can't or at least doesn't fill in everything and yet it is still valid to go to SME review